web-marketplace/src/types/overrides-mui.d.ts

```diff
@@ -0,0 +1,29 @@
+import { MuiPickersOverrides } from '@mui/lab/typings/overrides';
```

Comment:
maybe including the web-components existing file in web-marketplace/tsconfig.json as suggested in https://stackoverflow.com/a/75699722 would avoid us to repeat this twice?

Reply:
Good point will do that.